include/mbgl/util/interpolate.hpp

```diff
@@ -148,7 +148,7 @@ struct Interpolator<style::Rotation> {
 
 struct Uninterpolated {
     template <class T>
-    T operator()(const T& a, const T&, const double) const {
+    T operator()(const T& a, const T&, const double) const noexcept {
```

Comment:
```suggestion
    T operator()(const T& a, const T&, const double) const {
```

may call throwing copy constructor